spec/bullet/rack_spec.rb

```diff
@@ -60,15 +60,15 @@ module Bullet
         it "should return original response body" do
           expected_response = Support::ResponseDouble.new "Actual body"
           app.response = expected_response
-          _, _, response = middleware.call([])
+          _, _, response = middleware.call({})
```

Comment:
array argument is also valid, so don't replace it, add hash argument as a new test